doc/lightning-getroute.7.md

```diff
@@ -286,7 +286,6 @@ On success, an object containing **route** is returned.  It is an array of objec
 - **amount\_msat** (msat): The amount expected by the node at the end of this hop
 - **delay** (u32): The total CLTV expected by the node at the end of this hop
 - **style** (string): The features understood by the destination node (always "tlv")
-- **msatoshi** (u64, optional) **deprecated, removal in v23.05**
```

Comment:
from a doc point of view, do you think that it is better remove from the docs when it is already released? so in the v23.08 in this case

I think this could add a little bit of more documentation when people find this breaking change

Reply:
Hmm... that's a bit trickier, we'd have to add a "removed": "version" tag in the schema.  Probably not worth it for now?